src/components/Camera/style.css

```diff
@@ -103,6 +103,18 @@
   /*Mirroring transform: scale(-1, 1); */
 }
 
+/* this will only apply styles to Edge */
+@supports (object-fit: cover) and (-ms-ime-align:auto) {
```

Comment:
`@supports (object-fit: cover)` is redundant, no?

Reply:
Well, sort of. We know IE doesn't support `getUserMedia` but should they do in the future, this will still only target Edge.

Reply:
Unless they also implement  `object-fit: cover` ;)

Reply:
well, Edge is the new IE :P